### PR TITLE
docs: migrate from venus-docs

### PR DESCRIPTION
**Pull request by Fatman13**
fix https://github.com/filecoin-project/venus/issues/5019

### CODE REVIEW

**Comment by Fatman13** [comment]
请求 @simlecode 大神review。

File: docs/en/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+```
+
+### Run messager
+
+> ./venus-messager --config=xx.toml run [options]
+
+* Specify the directory of the configuration file by `--config=xxx.toml`，default: `./messager.toml`
+* If the specified configuration file does not exist at startup, a configuration file with the corresponding name will be generated and the value of the set parameter will be written to the configuration file
+* When the specified configuration file exists at startup, the values of the set parameters will be used, but will not be written to the configuration file. If the parameters are not set, the values of the parameters in the configuration file will be used
+
```
Comment (simlecode) [inline docs/en/getting-started.md:26]: 1.6.0 之后不需要配置 `--config=xx.toml`了，下面三个也可以删掉
* Specify the directory of the configuration file by `--config=xxx.toml`，default: `./messager.toml`
* If the specified configuration file does not exist at startup, a configuration file with the corresponding name will be generated and the value of the set parameter will be written to the configuration file
* When the specified configuration file exists at startup, the values of the set parameters will be used, but will not be written to the configuration file. If the parameters are not set, the values of the parameters in the configuration file will be used

File: docs/en/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+```bash
+options：
+  --auth-url           url for auth server
+  --auth-token         token for auth server
+  --node-url           url for connection lotus/venus
+  --node-token         token auth for lotus/venus
+  --db-type            which db to use. sqlite/mysql
+  --sqlite-path        sqlite db file. eg. ~/sqlite/message.db
```
Comment (simlecode) [inline docs/en/getting-started.md:34]: `--sqlite-path` 这个flag已经移除了

File: docs/zh/快速上手.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+make
+```
+
+### 启动messager
+
+> ./venus-messager --config=xx.toml run [command options]
+
+* 可以通过 `--config=xx.toml` 来指定配置文件的目录，默认目录是 `./messager.toml`
+* 启动时指定的配置文件不存在，则会生成对应名称的配置文件，并把设置的参数的值写到配置文件
+* 启动时指定的配置文件存在，设置的参数的值会被使用，但不会写到配置文件，未设置的参数则使用配置文件中参数的值
```
Comment (simlecode) [inline docs/zh/快速上手.md:25]: 一样改下

File: docs/zh/快速上手.md [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+
+```bash
+options:
+   --auth-url       auth服务的URL
+   --auth-token     auth服务的token
+   --node-url       lotus/venus 节点的URL
+   --node-token     auth服务的URL
+   --db-type        使用的数据库类型，sqlite 或者 mysql
+   --sqlite-file    sqlite 数据库的文件，例子：~/sqlite/message.db
```
Comment (simlecode) [inline docs/zh/快速上手.md:34]: `--sqlite-file` 可以删掉

**Review by simlecode — APPROVED** [review]

**Pull request merged**